src/package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "nativescript-imagepicker",
-    "version": "6.2.0",
+    "version": "6.2.1",
```

```suggestion
    "version": "6.3.0",
```
One of the merged PRs is a feature so we should bump the minor version